### ISSUE TITLE
make all dots circular

### ISSUE THREAD

**Issue by blobject** [agarick/agave#68]
consistency > feel

i j : . etc.

**Comment by zoomlogo** [comment]
Only the ASCII subset uses square dots right?

**Comment by blobject** [comment]
It's a bit complicated to answer because it depends on what we mean by "dot".

- If we only mean the tittles over `i` and `j`, then the answer is "basically yes". There are a few glyphs later on that reference them, like the "upside down i". There is also the miniscule `i`.

- If we only mean the dot shape used in punctuation, like `.`, `:`, `;`, there are several nooks and crannies. For instance, the `%` also currently uses square dots. The `,` is also square-y, as well as `'` and `"`. There might be a few more in ASCII. Then there are the punctuation-extensions, like "ellipses", "double exclamation mark", etc. Also a few math symbols like "division operator".

- If we mean it in a general stylistic sense, then right angles, dramatic angles, simplified edges and corners are ubiquitous.

A question might be, should I round off just the dots, or should I go and smooth out all the corners everywhere? So far, my reasoning has been to use square dots in the most common glyphs *in my use case*, and this has excluded diacritics and extended math symbols. I still haven't made a decision on this or looked at it too deeply.

**Comment by zoomlogo** [comment]
What do you mean by smooth out all corners?

Do you mean rounding the corners of all glyphs in Agave?

**Comment by blobject** [comment]
Yes, that's what I meant, as one end of the spectrum of possibilities. Since I've already smoothed some corners (because sharp angles everywhere can easily look terrible), yet retained sharpness in others, this requires some more deliberation. It'll likely be a line in the sand that I draw, whether an angle should be pointy or smoothed, I don't know yet.

**Comment by zoomlogo** [comment]
IMO, I think rounds the dots is better than changing the corners of **all** glyphs to round.

**Comment by alexmyczko** [comment]
the type is already round, i like the dots being square. i'd have to revisit it in Display/large mode. but mostly use it in the sizes 8-16 for terminals... not sure it'd still look Agave'y if it had round dots.
